include/z64audio.h

```diff
@@ -1006,17 +981,17 @@ typedef struct {
 } AudioContext; // size = 0x6450
 
 typedef struct {
-    /* 0x00 */ u8 reverb; // i.e. volume
+    /* 0x00 */ u8 reverbVol;
```

Comment:
is this actual reverb? (in the `NoteSubAttributes` struct)

Reply:
yep. It eventually gets passed to aEnvMixer